client/components/AbsoluteDate/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+
+/**
+ * Display absolute date in <time> element
+ *
+ * Usage:
+ * <AbsoluteDate date={historyItem._created} />
```

Comment:
hi @vied12,  The desired output to display timestamp for eventHistory is similar to what is displaying in article's itemHistory tab in SD(in preview pane) like `Wednesday, 28. June 13:28` or `Today, 30. June 12:06` and so on, so i tried AbsoluteDate component which is almost exact implementation of sdAbsdate directive(in /scripts/core/datetime/absdate-directive.js), used in itemHistory. whereas, DateTime comp. is giving config format. please suggest.